examples/SubscribeHelloWorldP2P/README.md

```diff
@@ -0,0 +1,28 @@
+# SubscribeHelloWorld example
+
+This example will show how to receive data from the DDS World creating a client subscriber.
```

Comment:
DDS World? Is not supposed to use the `ced` middleware?.
Update README with the right setup.

Reply:
Modified!